projects/metis/src/app/dataset/depublication/depublication.component.ts

```diff
@@ -56,9 +56,9 @@ export class DepublicationComponent extends DataPollingComponent {
   });
 
   formFile = this.fb.group({
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
     depublicationFile: [
-      (undefined as any) as File,
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      (undefined as unknown) as File,
```

Comment:
The linter complained about it.  The eslint comment wasn't taking effect because of the way that "prettier" split the line.  It's not a functional change, but a warning suppression.

Reply:
:+1: